demos/projects/ESPRESSIF/aziotkit/README.md

```diff
@@ -158,13 +157,13 @@ idf.py build
     On **Windows**:
 
     ```shell
-    idf.py -p COM5 flash
+    idf.py --no-ccache -B "C:\espbuild" -p COM5 flash
     ```
 
     On **Linux**:
 
     ```shell
-    idf.py -p /dev/ttyUSB0 flash
+    idf.py --no-ccache -B "C:\espbuild" -p /dev/ttyUSB0 flash

```

Comment:
This should be kept as is I think. The long path issue is only or mostly on Windows.
And if we verify it is an applicable workaround, the path should be something like `/espbuild` since it's Linux.

Reply:
let's keep as it is.